tests/utils/useUpdateAtom.test.tsx

```diff
@@ -1,6 +1,6 @@
-import React, { StrictMode, useEffect, useRef } from 'react'
```

Comment:
I would rather add `<StrictMode>` around `<Provider>`, if it doesn't break the test.

Reply:
done